test/python/quantum_info/operators/test_dihedral.py

```diff
@@ -205,7 +212,17 @@ def test_2_qubit_identities(self):
         circ2.cx(1, 0)
         elem1 = CNOTDihedral(circ1)
         elem2 = CNOTDihedral(circ2)
+
+        circcs01 = QuantumCircuit(2)
+        circcs01.cs(0, 1)
+        elemcs01 = CNOTDihedral(circcs01)
+
+        circcs10 = QuantumCircuit(2)
+        circcs10.cs(1, 0)
+        elemcs10 = CNOTDihedral(circcs10)
         self.assertEqual(elem1, elem2, "Error: 2-qubit CS identity does not hold")
+        self.assertEqual(elem1, elemcs01, "Error: 2-qubit CS identity does not hold")
+        self.assertEqual(elem1, elemcs10, "Error: 2-qubit CS identity does not hold")
```

Comment:
Just FYI. I personally like the style having `assert` right after each `elemX` is defined:
```
        elem2 = CNOTDihedral(circ2)
        self.assertEqual(elem1, elem2, "Error: 2-qubit CS identity does not hold")
        
        ...
        elemcs01 = CNOTDihedral(circcs01)
        self.assertEqual(elem1, elemcs01, "Error: 2-qubit CS identity does not hold")
        
        ...
```